docs/0-intended-use/index.md

```diff
@@ -219,53 +170,21 @@ specific servers.
 
+The {{device_name}} software is designed to fulfil the `Automated Contourer`
+role within the [IHE RO BRTO-II profile](http://ihe-ro.org/doku.php?id=doc:profiles:brto-ii).
+
+This is defined as:
+
+> A system that consumes [an image series] and creates an RT Structure Set. ...
+
 ### To be utilised within the framework of standard DICOM input and output
 
 The software is intended to receive DICOM files that conform to the DICOM
 standard. The software that it provides results to is intended to receive and
 appropriately handle standards compliant DICOM structure files. It is not
 intended to directly interface with another medical device except via this
-standard DICOM input output (either file based or DICOM network protocol based).
-
-It is by this intended use that criteria 1 for CDSS to be considered exempt is
-met:
-
-> **Criteria 1**: It is not intended to directly process or analyse a medical
-> image or a signal from another medical device (including an in vitro
-> diagnostic device).
-
-### Pseudonymisation key for patient header information
-
-Identifying patient header information is pseudonymised so that reversal is
-only achievable via a key that is never sent to the Radiotherapy AI cloud
-server. This key is stored within the Windows Credential manager vault within
-the operating system where the local Radiotherapy AI DICOM server is installed.
-
-This key is to be treated with the same level of protection as the patient
-information it is encrypting. The Windows Credential vault is encrypted by the
-OS using the Windows account login password. It is expected that both
-access to that account and its account password meets requirements appropriate
-to the protection of identifying patient information.
-
-### Validation of pseudonymisation undergone with a representative sample
-
-The Radiotherapy AI DICOM server can be configured to not send files to the
-cloud but to instead send the pseudonymised data either to file, or to another
-piece of software via a DICOM send. This utility is to be used after
-installation, and after any upgrades to the DICOM server that adjust the
-pseudonymisation to verify that the appropriate header items within the DICOM
-file have been appropriately de-identified by the installed server.
-
-### Not to receive DICOM images that have identifying patient information burned into the pixel data
-
-Only identifying patient information that is stored within the standards
-compliant DICOM header is pseudonymised before sending to the cloud sever.
-
-The DICOM software can be configured to only send through certain DICOM file
-types. By default this is CT, MRI and RT-DICOM specific files. This is to help
-prevent bulk DICOM sends containing other DICOM types that sometimes have burnt
-in pixel data included having those extra images included in the
-pseudonymisation + upload.
+standard DICOM input output (either file based or DICOM network protocol
```

Comment:
input and/or output?

Reply:
Fixed in d9fc69e62dcf8ed6eca258b7beb0e20c32bc031e